lib/di.dart

```diff
@@ -1188,5 +1198,63 @@ Future<void> setup({
   getIt.registerFactory(() => NFTViewModel(appStore, getIt.get<BottomSheetService>()));
   getIt.registerFactory<TorPage>(() => TorPage(getIt.get<AppStore>()));
 
+  getIt.registerFactory<LightningViewModel>(
+    () => LightningViewModel(),
+  );
+
+  getIt.registerFactory<LightningSendViewModel>(
+    () => LightningSendViewModel(
+      settingsStore: getIt.get<SettingsStore>(),
+      fiatConversionStore: getIt.get<FiatConversionStore>(),
+    ),
+  );
+
+  getIt.registerFactoryParam<LightningInvoicePageViewModel, void, void>((_, __) {
+    return LightningInvoicePageViewModel(
+      getIt.get<SettingsStore>(),
+      getIt.get<AppStore>().wallet!,
+      getIt.get<SharedPreferences>(),
+      getIt.get<LightningViewModel>(),
+    );
+  });
+
+  getIt.registerFactoryParam<LightningReceiveOnchainPage, void, void>((_, __) {
+    return LightningReceiveOnchainPage(
+      addressListViewModel: getIt.get<WalletAddressListViewModel>(),
+      lightningViewModel: getIt.get<LightningViewModel>(),
+      receiveOptionViewModel:
+          getIt.get<ReceiveOptionViewModel>(param1: lightning!.getOptionOnchain()),
+    );
+  });
+
+  getIt.registerFactoryParam<LightningInvoicePage, void, void>((_, __) {
+    return LightningInvoicePage(
+      lightningInvoicePageViewModel: getIt.get<LightningInvoicePageViewModel>(),
+      lightningViewModel: getIt.get<LightningViewModel>(),
```

Comment:
these 2 will have different instances of LightningViewModel, is this the intended behavior? I don't see a reason for it tho
actually it's unused